doc/README.md

```diff
@@ -1,11 +1,17 @@
 # ROS2 Navigation System Documentation
 This is where the ROS2 Navigation System documentation is being collected and vetted.
```

Comment:
Perhaps we should use "ROS 2 Navigation framework"

Reply:
We discussed and will leave as is for consistency